docs/assemblies/deploying.adoc

```diff
@@ -688,3 +688,34 @@ $ oc get openstackdataplanenodeset openstack-edpm-ipam -o json | jq '.status.con
   "type": "NodeSetDNSDataReady"
 }
 ----
+
+=== Dataplane deployment Pods
+
+
+During reconciliation of OpenStackDataPlaneDeployment resources,
+OpenStackAnsibleEE resources, marked with label `osdpd=<OpenStackDataPlaneDeployment.Name>,
+are created to manage the ansible executions that configure the EDPM nodes.
+The OpenStackAnsibleEE resources create Kubernetes Jobs resources, which in turn create Kubernetes Pods that are executing ansible.
+
+Checking the logs of these Jobs and Pods can show ansible failures and help diagnose issues.
+
+Pods launched during deployement throught OpenStackAnsiblEEE managed jobs,
+further referred to as pods, are are marked with label `osdpd=<OpenStackDataPlaneDeployment.Name>`
```

Comment:
drop "further referrred to as pods" clause

Reply:
ditto